projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -460,6 +476,28 @@ public static function deleted_plugin( $plugin_file, $deleted ) {
 		}
 	}
 
+	/**
+	 * REST prepare_item_for_response filter.
+	 *
+	 * @param array            $item    WP Cron event.
+	 * @param \WP_REST_Request $request Request object.
+	 * @return array Response array on success.
+	 */
+	public static function response_filter( $item, $request ) { // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.FoundAfterLastUsed, VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
Let's not pass a second argument to the filter callback if it ends up being unused.
The function name and docs are not very descriptive. What does it do?